src/common/objects/responseMessages.ts

```diff
@@ -17,6 +17,7 @@ export const RESPONSE_MESSAGE: {
   SIGNIN_USER_SUCCESS: '로그인/회원가입 성공',
   ISSUED_TOKEN_SUCCESS: '토큰 발급 성공',
   REISSUED_TOKEN_SUCCESS: '토큰 재발급 성공',
+  SIGNOUT_USER_SUCCESS: '회원탈퇴 성공',
```

Comment:
```suggestion
  WITHDRAW_USER_SUCCESS: '회원탈퇴 성공',
  or
  DELETE_USER_SUCCESS: '회원탈퇴 성공',
```
이부분 `SIGNOUT` 이 로그아웃 같고, 통상적으로 `withdraw` 라는 단어를 쓰는 것 같아 WITHDRAW 나 DELETE 로 바꾸는게 어떨까요?

Reply:
좋슴다 !